zimui/src/pages/HomePage.vue

```diff
@@ -19,21 +19,37 @@ watch(params, () => {
 
 // fetch channel data and set default topic if needed
 onMounted(async () => {
-  await main.fetchChannel()
-  if (topic.value === undefined && main.channelData != null) {
-    topic.value = main.channelData.rootSlug
+  try {
+    await main.fetchChannel()
+    if (topic.value === undefined && main.channelData != null) {
+      topic.value = main.channelData.rootSlug
+    }
+  } catch (error) {
+    console.error('Error fetching channel data:', error)
```

Comment:
Again, no console.log + all errors displayed on the UI

Reply:
@benoit74 sorry, I didn't mean to sound rude. I will try to produce the error by myself and provide you with the current styles of error messages. But to be honest I might need a tester to help me test this feature(It seems that it needs too much digging into the stuff to produce these errors). I would really appreciate if you could provide me with a test case or a tester.